build.gradle

```diff
@@ -53,12 +53,14 @@ apply plugin: 'idea'
 apply plugin: 'opensearch.opensearchplugin'
 apply plugin: 'opensearch.testclusters'
 apply plugin: 'opensearch.rest-test'
-
+apply plugin: 'checkstyle'
 
 checkstyle {
     toolVersion = '8.24'
     configFile file("checkstyle/checkstyle.xml")
 }
+checkstyleMain.ignoreFailures = true
+checkstyleTest.ignoreFailures = true
```

Comment:
Do we plan to enable checkstyles later?

Reply:
Yes.  We will enable checkstyleMain and checkstyleTest tasks later.

Reply:
Opensearch Core has removed CheckStyle and are using Spotless as the formatting plugin from now. Ref: https://github.com/opensearch-project/OpenSearch/issues/1362